platform/core/src/DataSources/IWebApiDataSource.js

```diff
@@ -54,9 +55,12 @@ function create({
     },
   };
 
+  const defaultReject = {};
+
   return {
     query: query || defaultQuery,
     retrieve: retrieve || defaultRetrieve,
+    reject: reject || defaultReject,
```

Comment:
Might be worth a small explanation about rejection as an inlined comment on the interface? Also curious if you have a good DCM4CHEE resource to link to on the different kinds of rejections, and how you configured DCM4CHEE to behave the way you would expect?